examples/tests/cloud-push-pull/azure-container-registry.earth

```diff
@@ -0,0 +1,12 @@
+FROM earthly/dind:alpine-main
+
+push:
+    RUN date | md5sum | head -c 32 > motd
+    ENTRYPOINT cat motd
+
+    SAVE IMAGE --push earthlyintegrationtest.azurecr.io/test:latest
+
+pull:
+    WITH DOCKER --pull earthlyintegrationtest.azurecr.io/test:latest
+        RUN docker run earthlyintegrationtest.azurecr.io/test:latest
+    END
```

Comment:
Theoretically a `FROM earthlyintegrationtest.azurecr.io/test:latest` would involve a pull just as well. Though this works too.

Reply:
I wanted to actually run it to have the hash I generated print. To ensure its functional.

Reply:
Yeah, makes sense.